package.json

```diff
@@ -91,6 +91,7 @@
     "bundlesize": "^0.18.0",
     "clean-webpack-plugin": "^3.0.0",
     "copy-webpack-plugin": "^5.0.4",
+    "copyfiles": "^2.3.0",
```

Comment:
This package seems to heavy for what we need, 
![image](https://user-images.githubusercontent.com/6320236/83564023-4a527d80-a4d9-11ea-8e4c-3e8502110fb3.png)

Comparing to npc:
![image](https://user-images.githubusercontent.com/6320236/83563977-3870da80-a4d9-11ea-94a7-edc2a6ef716a.png)

https://bundlephobia.com/result?p=copyfiles@2.3.0